ledger-service/jwt/BUILD.bazel

```diff
@@ -34,24 +38,13 @@ da_scala_library(
 
 da_scala_binary(
     name = "jwt-bin",
-    srcs = glob(["src/main/scala/**/*.scala"]),
     main_class = "com.daml.jwt.Main",
-    scala_deps = [
-        "@maven//:com_github_scopt_scopt",
-        "@maven//:com_typesafe_scala_logging_scala_logging",
-        "@maven//:org_scalaz_scalaz_core",
-    ],
     scalacopts = lf_scalacopts,
     runtime_deps = [
-        ":jwt",
         "@maven//:ch_qos_logback_logback_classic",
     ],
     deps = [
-        "//daml-lf/data",
-        "@maven//:com_auth0_java_jwt",
-        "@maven//:com_auth0_jwks_rsa",
-        "@maven//:com_google_guava_guava",
-        "@maven//:org_slf4j_slf4j_api",
+        ":jwt",
     ],
 )
```

Comment:
I am not sue why there is both the source 
```srcs = glob(["src/main/scala/**/*.scala"]),```
and the library
```
 ":jwt",
```

Reply:
looks broken, the library should exclude the `Main` class and then the binary should depend on the lib and the source for the Main class. But your change seems fine for now.